src/DependencyInjection/Configuration.php

```diff
@@ -11,7 +11,7 @@ final class Configuration implements ConfigurationInterface
 {
     public function getConfigTreeBuilder(): TreeBuilder
     {
-        $treeBuilder = new TreeBuilder('acme_sylius_example_plugin');
+        $treeBuilder = new TreeBuilder('webgriffe_sylius_table_rate_shipping_plugin');
```

Comment:
```suggestion
        $treeBuilder = new TreeBuilder('webgriffe_sylius_table_rate_shipping');
```
If i remember correctly you must not put the "_plugin" suffix here otherwise it will fail to load the configs.

Reply:
That's not completely true.. This string is not the same one used to read the configuration in your application, it is more like a "title". Check [here](https://github.com/webgriffe/SyliusActiveCampaignPlugin/blob/0968216a5045467da7416ac57b149355aa56413d/src/DependencyInjection/Configuration.php#L21) and [here](https://github.com/webgriffe/SyliusActiveCampaignPlugin/blob/0968216a5045467da7416ac57b149355aa56413d/tests/Application/config/packages/webgriffe_sylius_active_campaign_plugin.yaml#L4) for example..

Reply:
Ok, true!